resources/js/Layouts/FrontendLayout.tsx

```
@@ -185,15 +195,98 @@ const FrontendLayout = ({ children }: PropsWithChildren) => {
+      <main className="mx-auto w-[960px]">{children}</main>
 
-      <footer className="border-t border-gray-200 dark:border-gray-700 mt-12 py-4">
-        <div className="w-[960px] mx-auto text-sm text-gray-600 dark:text-gray-300">
-          &copy; {appName}
+      <footer className="mt-12 border-t border-gray-200 py-6 dark:border-gray-700">
+        <div className="mx-auto w-[960px]">
+          <div className="flex items-center justify-between">
+            <div className="w-full sm:w-1/2">
+              <p
+                className="text-sm text-gray-600 dark:text-gray-400"
+                dangerouslySetInnerHTML={{ __html: footerText }}
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)